ssstar/src/config.rs

```diff
@@ -74,7 +76,11 @@ pub struct Config {
         feature = "clap",
         clap(long, global = true, conflicts_with_all = &["aws_access_key_id", "aws_secret_access_key", "aws_session_token"])
     )]
-    pub aws_role_session_duration_seconds: Option<i32>,
+    pub aws_role_session_duration_seconds: Option<u64>,
+
+    /// Custom credentials provider, this is useful if you prefer providing credentials on yourself
+    /// which may expire from time to time.
+    pub credentials_provider: Option<CustomCredentialsProvider>,
```

Comment:
Is it possible to put smth like `#[clap(skip)]` here to remove the requirements for implementing `FromStr`, and `PartialEq`?

```suggestion
    #[clap(skip)]
    pub credentials_provider: Option<CustomCredentialsProvider>,
```

Reply:
it helps, now I don't need to implement `FromStr` but it still requires `PartialEq`, I think `clap` can't resolve the issue that it requires `PartialEq` because it doesn't related to clap, it's needed because we have a test where we compare the default variant of config with expected variant

Reply:
I see, makes sense, thank you for explanation